README.md

```diff
@@ -4,17 +4,59 @@
 [![Discord](https://img.shields.io/badge/chat-on%20disord-green.svg?logo=discord)](https://discord.gg/5E9zrhguTy)
 
 
-> 'JS Make' - parallel task runner for the frontend ecosystem with a JS extension system.
+Chomp is frontend task runner with advance featured focused on _ease-of-use_ and _not getting in the way_!
+
+## Why is Chomp easy to use?
```

Comment:
Rather than justifying being easy to use, I think this is "easy to switch to".

Perhaps something like:

```suggestion
## One-line migration from npm scripts
```